### ISSUE TITLE
Deprecation warning when using --progress

### ISSUE THREAD

**Issue by nifgraup** [webpack/webpack-cli#288]
**Do you want to request a *feature* or report a *bug*?** bug / deprecation warning

**What is the current behavior?**
webpack-cli makes use of a deprecated API in module tapable

**If the current behavior is a bug, please provide the steps to reproduce.**

```
$ npm install webpack webpack-cli
+ webpack-cli@2.0.9
+ webpack@4.0.0

$ mkdir src
$ echo > src/index.js
$ node node_modules/webpack-cli/bin/webpack.js --mode development --progress
  0% compiling(node:25726) DeprecationWarning: Tapable.apply is deprecated. Call apply on the plugin directly instead
```
**What is the expected behavior?**
no warning

**Comment by ev1stensberg** [comment]
@TheLarkInn ?

**Comment by erikdesjardins** [comment]
This is fixed by #305, but it seems it hasn't been released
